Makefile.am

```diff
@@ -81,6 +81,8 @@ all-local: digraphs.la
 	cp -RL @PLANARITY_SUITE_DIR@/.libs/*  $(top_srcdir)/bin/lib/
 if SYS_IS_CYGWIN
 	cp .libs/digraphs.dll $(GAPINSTALLLIB)
+# Cygwin will only look in this directory for dlls
+	cp bin/libs/cygplanarity-0.dll $(GAPROOT)/.libs
```

Comment:
I need to check whether this should actually be `libs` or if it should be `lib` (I can't work out where the "s" would come from, but I wrote "libs" in my comments on #177).

Reply:
it's bin/lib, no s